Documentation/using-prerelease-packages.md

```diff
@@ -0,0 +1,61 @@
+# Using Pre-Release Packages
+
+Cesium for Unreal has a continous integration (CI) system that automatically builds an installable version of the plugin for every commit in every branch. You can download and install these yourself to try out new features before they're released.
+
+_Before you begin, **back up your project**. Cesium for Unreal can not guarantee that levels saved with a pre-release version will be loadable in other versions of the plugin._
+
+* [Download a pre-release version from a Pull Request](#download-a-pre-release-version-from-a-pull-request)
+* [Download a pre-release version from a Branch or Commit](#download-a-pre-release-version-from-a-branch-or-commit)
+
+## Download a pre-release version from a Pull Request
+
+To download a pre-release version built from a pull request, click the "Checks" tab at the top of the pull request:
+
+<img width="601" alt="image" src="https://github.com/CesiumGS/cesium-unreal/assets/924374/4e942591-b1b2-4d9d-9892-3ff05922ce48">
```

Comment:
@kring If you don't mind the recapture, can I recommend using another tool to highlight the area of interest to make it look more polished?

Reply:
Committed some changes to polish those yellow circles. (used Windows Paint 3D, 2D shapes, circle tool)